### ISSUE TITLE
Create a report request causes fetch to blow up (maybe)?

### ISSUE THREAD

**Issue by domfarolino** [WICG/attribution-reporting-api#546]
https://wicg.github.io/attribution-reporting-api/#create-a-report-request creates a new request whose [origin](https://fetch.spec.whatwg.org/#concept-request-origin) is "client" (by default) and whose client is "null" (explicitly).

But the 10th step of [fetch](https://fetch.spec.whatwg.org/#concept-fetch) is:

> If request’s [origin](https://fetch.spec.whatwg.org/#concept-request-origin) is "client", then set request’s [origin](https://fetch.spec.whatwg.org/#concept-request-origin) to request’s [client](https://fetch.spec.whatwg.org/#concept-request-client)’s [origin](https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-origin).

But since client is "null", it seems this step would fail. It's a little late as I'm looking at this, so I apologize if I am missing something.

**Comment by apasel422** [comment]
Thanks for pointing this out. I think you're right that the draft specification's setting of these two properties is inconsistent.

If you have suggestions for how to resolve it, feel free to create a PR.

The most obvious value to use as the request's origin would be the reporting origin itself, i.e. the origin the report is being sent to.

**Comment by domfarolino** [comment]
Yeah I think setting the origin explicitly is what will be needed. The request's same-origin-ness will be computed based off of that.
